bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdObservation2.java

```diff
@@ -582,7 +582,7 @@ private void _showStatus() {
         final boolean override = getDataObject().isOverrideQaState();
 
         // Format the dataset disposition ("dataflow step").
-        final String statusString = (dispo.isEmpty()) ? "No Data" : dispo.get().description();
+        final String statusString = (dispo == null || dispo.isEmpty()) ? "No Data" : dispo.get().description();
```

Comment:
Since the `lookupDatasetDisposition` method already returns an `Option`:

````scala
final scala.Option<DataflowStatus> dispo = DatasetDispositionService.lookupDatasetDisposition(getNode());
````

we might want to make it return `None` instead of `null` for this case (no obslog).